app/src/main/java/com/termux/x11/LoriePreferences.java

```diff
@@ -142,6 +142,14 @@ void updatePreferencesLayout() {
                     Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU
                     && ContextCompat.checkSelfPermission(requireContext(), POST_NOTIFICATIONS) == PERMISSION_DENIED;
             findPreference("requestNotificationPermission").setVisible(requestNotificationPermissionVisible);
+
+
+
+            findPreference("displayScale").setVisible(true);
```

Comment:
this should remove?

Reply:
Hi I fixed the commit removing that unwanted change, not sure how it got there.